### DISCUSSION TITLE
WinstaNext V1.0.0.0

### DISCUSSION THREAD

**Discussion by NGame1** [NGame1/Winsta11#14] [Show and tell]
WingoTag has now reached our first Milestone.
- Added support for Ctrl + V on Directs thread
- Added coAuthor and Hashtag support to posts
- Better URL support in directs, captions and etc.
- Fix for hashtags follow button content and behavior
- Other fixes and improvements.

<hr /><em>This discussion was created from the release <a href='https://github.com/NGame1/Winsta11/releases/tag/Next1.0.0.0'>WinstaNext V1.0.0.0</a>.</em>

**Comment by syler1849** [comment]
Hello I need to talk to you about InstagramApiSharp for my project, contact me $$$$ WhatsApp +5541998009937
